server/models/bhima.sql

```diff
@@ -90,14 +90,14 @@ INSERT INTO unit VALUES
   (203,'Employees Configuration','TREE.PAYROLL_EMP_CONFIGURATION','',57,'/payroll/employee_configuration'),
   (204,'Exchange Rate','TREE.EXCHANGE','',1,'/exchange'),
   (205,'Account Reference Management','TREE.ACCOUNT_REFERENCE_MANAGEMENT','',1,'/account_reference'),
-  (206,'[OHADA] Bilan','TREE.OHADA_BALANCE_SHEET','',281,'/reports/ohada_balance_sheet_report'),
+  (206,'OHADA Bilan','TREE.OHADA_BALANCE_SHEET','',281,'/reports/ohada_balance_sheet_report'),
```

Comment:
Don't these changes need to be added to the migrate.sql file too?
I realize that using the translation item (eg, 'TREE.OHADA_BALANCE_SHEET') fixes this in the UI automatically since it is changed in the *.json files.  But if the second item in the 'unit' rows are ignored, why do we have them?

Reply:
Good question.  We have them so that we know what they are in the database when we are doing `SELECT * FROM unit;`.  Otherwise, there isn't anything that tells us what that unit is.